src/test/javascript/spec/component/competencies/competency-form.component.spec.ts

```diff
@@ -130,29 +122,33 @@ describe('CompetencyFormComponent', () => {
     });
 
     it('should suggest taxonomy when title changes', () => {
-        const suggestTaxonomySpy = jest.spyOn(competencyFormComponent, 'suggestTaxonomies');
-        const translateSpy = createTranslateSpy();
         competencyFormComponentFixture.detectChanges();
 
+        const commonCourseCompetencyFormComponent = competencyFormComponentFixture.debugElement.query(By.directive(CommonCourseCompetencyFormComponent)).componentInstance;
+        const suggestTaxonomySpy = jest.spyOn(commonCourseCompetencyFormComponent, 'suggestTaxonomies');
+        const translateSpy = createTranslateSpy();
+
         const titleInput = competencyFormComponentFixture.nativeElement.querySelector('#title');
         titleInput.value = 'Building a tool: create a plan and implement something!';
         titleInput.dispatchEvent(new Event('input'));
 
         expect(suggestTaxonomySpy).toHaveBeenCalledOnce();
         expect(translateSpy).toHaveBeenCalledTimes(12);
-        expect(competencyFormComponent.suggestedTaxonomies).toEqual(['artemisApp.competency.taxonomies.REMEMBER', 'artemisApp.competency.taxonomies.UNDERSTAND']);
+        expect(competencyFormComponent.suggestedTaxonomies).toEqual(['artemisApp.courseCompetency.taxonomies.REMEMBER', 'artemisApp.courseCompetency.taxonomies.UNDERSTAND']);
```

Comment:
Once I remove one of the `merge`s in the CommonCourseCompetencyFormComponent, the test no longer fails - but this assertion fails. In fact, it seems that `CourseCompetencyFormComponent.suggestedTaxonomies` isn't used at all outside of the tests.

Should this be `commonCourseCompetencyFormComponent.suggestedTaxonomies` instead? The test passes for me if I make that change.